inst/utils/test-golem-recommended.R

```diff
@@ -24,3 +24,19 @@ test_that(
     golem::expect_running(sleep = 5)
   }
 )
+
+
+# Configure this test to fit your need. 
+# testServer() function makes it possible to test code in server functions and modules, without needing to run the full Shiny application
+testServer(app_server, {
```

Comment:
@Swechhya do you think you can move this be `test_that("app launches"` please? 

That would allow to keep the server test together :) 

For the content, it's perfect. That's a good skeleton for developers

Reply:
@ColinFay  I'm sorry but I'm not sure what change is required. Do you mean that we move this to a different file altogether?

Reply:
Nop, just move the code a couple of lines up, just so that the two `server` related test are onee after the other :)

Reply:
Ah! I understand now.